docs/configurations.md

```diff
@@ -0,0 +1,266 @@
+| scalar.db.jdbc.admin.connection_pool.max_total          | maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for admin. Use a negative value for no limit.              | 25                           |
+
+When you use SQLite3 as a JDBC database, you need to set `scalar.db.contact_points` as follows.
+
+```properties
+scalar.db.contact_points=jdbc:sqlite:your-db.sqlite3?busy_timeout=10000
+```
+
+Unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html).
+In order to avoid frequent errors caused internally by the [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy),
+it is recommended to set a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+
+##### Multi-storage
+
+ScalarDB supports a Multi-storage where you can use multiple storage implementations at the same time.
+You can use Multi-storage by specifying `multi-storage` for the `scalar.db.storage` property.
+
+For the details of the Multi-storage, see [Multi-storage Transactions](multi-storage-transactions.md).
+
+### ScalarDB Server (gRPC)
+
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to Scalar DB.
+To interact with ScalarDB Server, you need to specify `grpc` for the `scalar.db.transaction_manager` property.
+
+The following configurations are available for ScalarDB Server:
+
+| name                                                    | description                                               | default                |
+|---------------------------------------------------------|-----------------------------------------------------------|------------------------|
+| scalar.db.transaction_manager                           | `grpc` should be specified.                               | -                      |
+| scalar.db.contact_points                                | ScalarDB Server host.                                     |                        |
+| scalar.db.contact_port                                  | Port number for ScalarDB Server.                          | 60051                  |
+| scalar.db.grpc.deadline_duration_millis                 | The deadline duration for gRPC connections.               | 60000 (60 seconds)     |
+| scalar.db.grpc.max_inbound_message_size                 | The maximum message size allowed for a single gRPC frame. | The gRPC default value |
+| scalar.db.grpc.max_inbound_metadata_size                | The maximum size of metadata allowed to be received.      | The gRPC default value |
+
+For the details of ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
+
+### JDBC transactions
```

Comment:
I think there are several notes for `JDBC transactions` as follows.

1. You can use `JDBC transactions` with the JDBC database only (i.e., we cannot use `JDBC transactions` with some NoSQL databases).
2. You cannot use `JDBC transactions` with `Multi-storage transactions` even if the all underlying databases are JDBC databases.

Especially, I think the second one (limitation for `Multi-storage transactions`) is a bit confusing. So, I think it would be better to add a note for `Multi-storage transactions` in this section.

What do you think?

Reply:
Actually, as described in `configurations.md`, the storage property `scalar.db.storage ` is specifically for the Consensus Commit transaction manager. So other transaction manager implementations (`jdbc` and `grpc`) do not use it.

Previously, this was not explicitly mentioned, but in the updated `configurations.md` included in this PR, I think we have made this clear. Given this, do you think we still need to add the special note in the multi-storage transaction document?

Reply:
Thank you for the explanation! I understood the `scalar.db.storage` is a dedicated configuration for Consensus Commit.

I didn't notice the `scalar.db.storage` is a specific configuration for Consensus Commit. 

So, I think the special note is not necessary.

By the way, I was just curious, but if we specify both `scalar.db.transaction_manager=jdbc` and `scalar.db.storage=mutli-storage`, does ScalarDB ignore the values of `scalar.db.storage`?

Reply:
Not for this PR, but I think it is probably better to throw an exception if `scalar.db.transaction_manager=jdbc` and `scalar.db.storage` are both set.

Reply:
> By the way, I was just curious, but if we specify both scalar.db.transaction_manager=jdbc and scalar.db.storage=mutli-storage, does ScalarDB ignore the values of scalar.db.storage?

@kota2and3kan Basically, that's correct. But, there is an exception for this. I will explain it along with my response to @feeblefakie's comment.

> Not for this PR, but I think it is probably better to throw an exception if scalar.db.transaction_manager=jdbc and scalar.db.storage are both set.

@feeblefakie @kota2and3kan This is also related to @kota2and3kan's question.

In fact, there is a scenario where both configurations are used. This happens when you use both the JDBC transaction manager and the storage API directly. Although this seems like a very rare case, it's technically possible. While the JDBC transaction manager doesn't use the `scalar.db.storage` property, `scalar.db.storage` is used by the storage API.

Reply:
@brfrn169  
Ah, I see. Thank you for your answers!
I feel it is a little bit confusing for users... 
So, it might be better to separate properties for `DistributedTransaction` and `DistributedStorage` explicitly in the future.
(But, basically, users don't use Storage API directly. So, it might be OK to keep the current behavior.)